packages/kit/src/views/Staking/components/UniversalWithdraw/index.tsx

```
@@ -57,9 +62,14 @@
   onConfirm?: (amount: string) => Promise<void>;
 };
 
+const isNaN = (num: string) =>
```

Biome

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)